### PR TITLE
Fix crash on Safari

### PR DESCRIPTION
**Pull request by JosephMarinier**
## Description:

Fix crash on Safari and iOS browsers as they don't support lookbehind is regular expressions.

## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge it.

* [x] **ISSUE NUMBER.** You linked the issue number (Ex: Resolve #XXX).
* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

**Review by christyler3030 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,5 +21,6 @@ Released changes are shown in the
 ### Fixed
 - Showing long utterances fully on hover in similar and perturbed utterances tables.
 - Webapp crash when no pipeline.
+- Crash on Safari and iOS browsers as they don't support lookbehind is regular expressions.
```
Comment (lindsaydbrin) [inline CHANGELOG.md:24]:
```suggestion
- Crash on Safari and iOS browsers as they don't support lookbehind as regular expressions.
```

File: webapp/src/utils/format.ts [hunk 2]
```diff
@@ -13,4 +13,5 @@ export const formatRatioAsPercentageString = (
 ) => `${formatNumberAsString(100 * value, digits)}%`;
 
 export const camelToTitleCase = (camelCase: string) =>
-  camelCase.replace(/(?<=[a-z])(?=[A-Z0-9])|(?<=[A-Z0-9])(?=[A-Z][a-z])/g, " ");
+  // Safari and iOS browsers don't support lookbehind is regular expressions.
```
Comment (lindsaydbrin) [inline webapp/src/utils/format.ts:16]:
```suggestion
  // Safari and iOS browsers don't support lookbehind as regular expressions.
```

File: CHANGELOG.md (hunk 1 above)
Comment (JosephMarinier) [reply CHANGELOG.md:24]: I meant to write "in". Is that OK?
```suggestion
- Crash on Safari and iOS browsers as they don't support lookbehind in regular expressions.
```

Comment (lindsaydbrin) [reply CHANGELOG.md]: Yes! Better than my suggestion. :)

**Pull request merged**